CHANGELOG.md

```diff
@@ -41,7 +41,7 @@ To learn more about our roadmap, we recommend reading [this document](ROADMAP.md
 
 ### Improvements
 
-- TODO ([#XXX](https://github.com/kedacore/keda/issues/XXX))
+- **Event Hub Scale:** Support Pod Identity for Storage Blobs ([#3569](https://github.com/kedacore/keda/issues/3569))
```

Comment:
```suggestion
- **Event Hubs Scaler:** Support Azure AD Pod & Workload Identity for Storage Blobs ([#3569](https://github.com/kedacore/keda/issues/3569))
```

Reply:
Thank you for this change. I had this in my mind at some point but completely forgot.

Reply:
@v-shenoy I'm not sure that suggestion is correct as it seems to be solely around pod identity?

Reply:
PTAL at the comment I made below.

Reply:
LGTM, thanks!

Reply:
I'll test once when by evening today when I get time.